src/main/java/me/jellysquid/mods/sodium/client/IWorldRenderer.java

```diff
@@ -0,0 +1,7 @@
+package me.jellysquid.mods.sodium.client;
+
+import me.jellysquid.mods.sodium.client.render.SodiumWorldRenderer;
+
+public interface IWorldRenderer {
```

Comment:
Avoid using C# style prefixes on interface names. For example, a better name would be `WorldRendererAccess`.